.github/workflows/release.yml

```
@@ -36,44 +38,47 @@ jobs:
       - name: 🐙 Create GitHub Release
         uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
+          tag_name: ${{ github.ref_name }}
+          release_name: Release ${{ github.ref_name }}
           body: |
             ## 🔄 Backend Release
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)